java/gui/src/main/java/com/squareup/subzero/actions/InitWallet.java

```diff
@@ -66,7 +66,7 @@
 
       // Check if we need to create the pub key encryption key (only needs to happen once, on one machine)
       if (subzero.createPubKeyEncryptionKey) {
-        nCipher.createPubKeyEncryptionKey();
+        nCipher.createPubKeyEncryptionKeyTicket();
```

Comment:
is this intentional?

Reply:
Yes, a rename happened a while ago (commit was squashed) and I missed this spot.